src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlStream.cs

```diff
@@ -450,8 +379,11 @@ override public int Read(byte[] buffer, int offset, int count)
                 cb = _cachedBytes[_currentArrayIndex].Length - _currentPosition;
                 if (cb > count)
                     cb = count;
+#if NETFRAMEWORK
```

Comment:
We can just use one for both frameworks. Let's use `Buffer.BlockCopy` since it's a byte[].